packages/components/src/components.d.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable */
+/* tslint:disable */
+/**
+ * This is an autogenerated file created by the Stencil compiler.
+ * It contains typing information for all components that exist in this project.
+ */
+import { HTMLStencilElement, JSXBase } from "@stencil/core/internal";
+export namespace Components {
+    interface UiHeading {
+        "text": string;
+    }
+}
+declare global {
+    interface HTMLUiHeadingElement extends Components.UiHeading, HTMLStencilElement {
+    }
+    var HTMLUiHeadingElement: {
+        prototype: HTMLUiHeadingElement;
+        new (): HTMLUiHeadingElement;
+    };
+    interface HTMLElementTagNameMap {
+        "ui-heading": HTMLUiHeadingElement;
+    }
+}
+declare namespace LocalJSX {
+    interface UiHeading {
+        "text"?: string;
+    }
+    interface IntrinsicElements {
+        "ui-heading": UiHeading;
+    }
+}
+export { LocalJSX as JSX };
+declare module "@stencil/core" {
+    export namespace JSX {
+        interface IntrinsicElements {
+            "ui-heading": LocalJSX.UiHeading & JSXBase.HTMLAttributes<HTMLUiHeadingElement>;
+        }
+    }
+}
```

Comment:
After running the build, I got new changes here. As far as I know, this file should be committed with the latest changes, so don't forget to run `npm run build` before pushing the new updates.